CartManagement/cart.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="cart.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+    <title>Venice | ProductList</title>
+</head>
+<body>
+    <div class="flex-container flex-columns">
+    <nav class="nav">
+        <div class="left">
+        <a href="#" class="logo"><img src="/Images/venice-logo.png"></a>
+        <a href="#" class="btn btn-link brand">Venice</a>
+        </div>
+        <div class="right">
+        <div class="input-group search"><input type="text" placeholder="Search"></div>
+        <button class="btn btn-primary">Login</button>
+        <div class="badge">
+            <a><i class="fas fa-heart"></i>
+            <div class="badge-icon">0</div></a>
+        </div>
+        <div class="badge cart-icon">
+            <a><i class="fas fa-shopping-cart"></i>
+            <div class="badge-icon">0</div></a>
+        </div>
+    </div>
+    </nav>
+    <div class="flex-container">
+        <main class="ml mt flex-container flex-columns flex-center">
+            <h2 class="brand">My Cart<span class="text-small">(2)</span></h2>
+            <div id="card-container" class="mt flex-rows">
```

Comment:
Try to use class instead of id

Reply:
Thank you for pointing it out. i will keep that in mind from next time